app/views/teacher_training_adviser/steps/_uk_address.html.erb

```diff
@@ -1,8 +1,5 @@
-<% @page_title = "What is your address?" %>
+<% @page_title = "What is your postcode?" %>
 
-<%= f.govuk_fieldset legend: { text: 'What is your address?' } do %>
-  <%= f.govuk_text_field :address_line1, width: 20 %>
-  <%= f.govuk_text_field :address_line2, width: 20 %>
-  <%= f.govuk_text_field :address_city, width: 20 %>
-  <%= f.govuk_text_field :address_postcode, width: 20 %>
+<%= f.govuk_text_field :address_postcode, width: 20, label: { text: @page_title, size: "m", tag: "h1" } do %>
+  <p>We try and match you with an adviser providing support in your region, or based on the subject you’re interested in teaching.</p>
```

Comment:
```suggestion
  <p>We try and match you with an adviser providing support in your region.</p>
```
@EllieNodder Does talking about subject here sound a bit confusing? Is it sufficient to leave as just talking about region, given that we say 'try'?

Reply:
NB This screen also comes up in the returners flow

Reply:
@gemmadallmandfe wasn't sure if we needed to add the subject bit, given that 'We try and and match you with an adviser providing support in your region' won't always be true if, for ETAs, we're actually mostly trying to match them by subject?  Let's discuss with Andy and am happy to change if you both think we should get rid of the subject bit?